README.md

```diff
@@ -15,20 +15,21 @@
 - use intention revealing names
 - write self documenting code
 - don't leave [broken windows](https://blog.codinghorror.com/the-broken-window-theory/) unrepaired
- 
+
 ## Code Reviews
 Pending code reviews represent blocked threads of execution. To prevent getting blocked for more time than is necessary, follow these basic principles (inspired by [Glen Stanford](https://medium.com/@9len/on-code-review-16ea85f7c585)):
 - make code reviews your top priority
-- make your pull requests a pleasure to read 
+- make your pull requests a pleasure to read
 
 ## Quality
 
 - prioritize quality over speed | [the only way to go fast is to go well](http://butunclebob.com/ArticleS.UncleBob.VehementMediocrity)
 - [test, automate, test, automate](docs/automate-tests.md)
+- Tend your [test garden](docs/automate-tests.md#tend-your-test-garden)
```

Comment:
this is incredibly nit-picky: we've been using a first lowercase letter in the bullet point items of the list, mind changing it to match?

Reply:
hahaha no i don't mind hooray for consistency!